packages/grafana-ui/src/components/index.scss

```diff
@@ -1,6 +1,7 @@
 @import 'CustomScrollbar/CustomScrollbar';
 @import 'DeleteButton/DeleteButton';
 @import 'ThresholdsEditor/ThresholdsEditor';
+@import 'Table/TableInputCSV';
```

Comment:
Maybe this should be TableInputCSV as well?

Reply:
Do you mean, it should be in a folder named 'TableInputCSV'?  I imagined that all table things would go in the same folder.  Other components I imagine in that folder would be:

* Table -- show a single table
* Tables -- show table[] with a picker to switch them (like sheets tab)
* TableInputCSV -- this component
* TableEditor -- each cell is an input field

when I list them out, I think it makes sense to give each its own folder :) or perhaps group them together in a table folder?

Right now, `ColorPicker` has a few related components in the same folder

Reply:
ah, ok. I see. that's ok then.